api/seedreconfig/seedreconfig.go

```diff
@@ -56,6 +56,11 @@ type SeedReconfiguration struct {
 	// being upgraded. In the case of IBI, this material is generated when the
 	// cluster's kubeconfig is being prepared in advance.
 	KubeconfigCryptoRetention KubeConfigCryptoRetention
+
+	// RawNMStateConfig contains nmstate configuration YAML file provided as string.
+	// String will be written to file and will be applied with "nmstatectl apply" command.
+	// Example of nmstate configurations can be found in this link https://nmstate.io/examples.html
+	RawNMStateConfig string `json:"raw_nm_state_config,omitempty"`
```

Comment:
Is this only meant for IBI? We should probably emphasize this is not meant for IBU if so

Reply:
Sort of can be provided in IBU process but not sure how it will be done. Easier to get nmconnection files